prototype/p2p/network_lifecycle.go

```diff
@@ -0,0 +1,61 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"strings"
+)
+
+func (g *P2PModule) configure(protocol, address, external string, peers []string) {
+	g.protocol = protocol
+	g.address = address
+	g.externaladdr = external
+	g.peerlist = types.NewPeerlist()
+
+	// this is a hack to get going no more no less
+	for i, p := range peers {
+		peer := types.NewPeer(uint64(i+1), p)
+		port := strings.Split(peer.Addr(), ":")
```

Comment:
Doesn't strings.split return an array rather than just a single port value? The var name isn't intuitive.